test/unit/source.js

```diff
@@ -198,17 +198,18 @@ describe('Sources', function () {
         it('should instance and use FileSource with url', function () {
             const source = new FileSource({
                 url: '..',
+                parsedData: {},
```

Comment:
If you add this, you are doing the "should instance and use FileSource with parsedData" test below.

Reply:
I will change it to fetch a reel file.

Reply:
Ok

Reply:
`done()`